pages/k8s/gpu-workers.md

```diff
@@ -13,96 +13,101 @@ layout: [base, ubuntu-com]
 toc: False
 ---
 
-**Charmed Kubernetes** supports GPU-enabled
-instances for applications which can use them. The kubernetes-worker charm will
-automatically detect NVIDIA hardware and enable the appropriate support.
-However, the implementation of GPU-enabled instances differs greatly between
-public clouds. This page outlines the recommended methods for running GPU
-enabled hardware for different public clouds.
+**Charmed Kubernetes** supports GPU-enabled instances for applications that
```

Comment:
in a strictly grammatical sense, either 'which' or 'that' are correct in this context, however we tend to prefer which for restrictive clauses as it also applies to non-restrictive clauses as well and can only really be misused when applied to people.

Reply:
:broom: